core/src/main/java/com/gentics/mesh/dagger/SearchWaitUtilProvider.java

```diff
@@ -0,0 +1,19 @@
+package com.gentics.mesh.dagger;
+
+import com.gentics.mesh.etc.config.MeshOptions;
+import com.gentics.mesh.event.MeshEventSender;
+import com.gentics.mesh.util.SearchWaitUtil;
+import com.gentics.mesh.util.SearchWaitUtilImpl;
+import dagger.Module;
+import dagger.Provides;
+
+import javax.inject.Singleton;
+
+@Module
+public class SearchWaitUtilProvider {

```

Comment:
Since this is implemented as a module, I'd ask  to rename it into `SearchWaitUtilProviderModule` to follow the existing naming convention.